jwst/pipeline/calwebb_spec2.py

```diff
@@ -307,9 +307,13 @@ def process_exposure_product(
             # wavelength bands
 
             resampled = calibrated.copy()
+            resampled = self.pixel_replace(resampled)
             resampled = self.cube_build(resampled)
             if not self.cube_build.skip:
                 self.save_model(resampled[0], suffix='s3d')
+        elif exp_type in ['MIR_LRS-SLITLESS']:
```

Comment:
Just a question out of curiosity: why does this mode not go through resample_spec like fixed-slit and MSA modes?

Reply:
MIRI LRS Slitless is a time-series observing mode and we never resample time series data from any instrument. They want high precision in relative signal across integrations and hence don't want to "muck with" the data any more than they have to.